benchmark/single-source/SetTests.swift

```diff
@@ -334,6 +610,98 @@ public let SetTests = [
     tags: [.validation, .api, .Set],
     setUpFunction: { blackHole([setP, setQ]) }),
 
+  BenchmarkInfo(
+    name: "Set.subtracting.Sequence.Empty.Int",
+    runFunction: { n in run_SetSubtractingSequenceInt(setE, arrayAB, 0, 1000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setE, arrayAB]) }),
+  BenchmarkInfo(
+    name: "Set.subtracting.Sequence.Int.Empty",
+    runFunction: { n in run_SetSubtractingSequenceInt(setAB, arrayE, countAB, 1000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, arrayE]) }),
+  BenchmarkInfo(
+    name: "Set.subtracting.Sequence.Empty.Box",
+    runFunction: { n in run_SetSubtractingSequenceBox(setOE, arrayOAB, 0, 1000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOE, arrayOAB]) }),
+  BenchmarkInfo(
+    name: "Set.subtracting.Sequence.Box.Empty",
+    runFunction: { n in run_SetSubtractingSequenceBox(setOAB, arrayOE, countAB, 1000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOAB, arrayOE]) }),
+  BenchmarkInfo(
+    name: "Set.Subtracting.Sequence.Int0",
```

Comment:
Lowercase `s`, same as above, `subtracting` is method name.
```suggestion
    name: "Set.subtracting.Seq.Int0",
```
...and same for similar cases below 👇